amundsen_application/api/metadata/v0.py

```diff
@@ -56,9 +56,14 @@ def popular_tables() -> Response:
     https://github.com/lyft/amundsenmetadatalibrary/blob/master/metadata_service/api/popular_tables.py
     """
     try:
+        if app.config['AUTH_USER_METHOD'] and app.config['POPULAR_TABLE_PERSONALIZATION']:
+            user_id = app.config['AUTH_USER_METHOD'](app).user_id
+        else:
+            user_id = ''
+
         service_base = app.config['METADATASERVICE_BASE']
         count = app.config['POPULAR_TABLE_COUNT']
-        url = f'{service_base}{POPULAR_TABLES_ENDPOINT}?limit={count}'
+        url = f'{service_base}{POPULAR_TABLES_ENDPOINT}/{user_id}?limit={count}'
```

Comment:
@feng-tao for submitting approval since you have full context on the metadata support. I took a look and see no issues as long as this endpoint creation logic is correct.